src/app/modules/image-viewer/image-viewer-layers/image-viewer-layers.component.spec.ts

```diff
@@ -4,11 +4,10 @@ import { ImageViewerLayer } from '../../../core/models/image-viewer-layer';
 import { ColorScheme } from '../../../modules/color-scheme/color-schemes';
 import { ImageViewerLayersComponent } from './image-viewer-layers.component';
 import { ImageViewerLayersModule } from './image-viewer-layers.module';
-import { async } from '@angular/core/testing';
 
 function getTestLayers(): ImageViewerLayer[] {
   const layers: ImageViewerLayer[] = [
-    {
+    new ImageViewerLayer({
```

Comment:
Sonar does not like the repeated new ImageViewLayer thing. Use a [{}, {}].map(l => new ImageViewLayer(l)) pattern to remove code duplication.